index.css

```diff
@@ -11,84 +12,88 @@
   }
 }
 
-.v2-alert {
-  padding: 20px 20px 20px 86px;
+@mixin icon($name) {
+  @include v2-icon($name, $color: "white", $after: true);
+}
+
+.alert {
+  display: inline-block;
   position: relative;
-  min-height: 50px;
-  font-size: 16px;
+  min-height: 52px;
   overflow: hidden;
-  background-color: #fff;
-  border-radius: 3px;
 }
 
-.v2-alert::after {
+.alert::after {
   /* use nib-styles/v2-icons as font to get icons */
   font: normal normal normal 14px/1 'nibdings';
   text-rendering: auto;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-  font-size: 40px;
+  font-size: 28px;
   position: absolute;
-  padding-top: 10px;
-  top: 15px;
-  bottom: 15px;
-  left: 0;
-  width: 65px;
-  color: #fff;
+  padding-top: 16px;
+  top: -4px;
+  left: -4px;
+  width: 60px;
+  min-height: 100%;
   text-align: center;
-  animation: v2-alert-pop-out 0.6s cubic-bezier(0.19, 1, 0.22, 1);;
 }
 
-.v2-alert--error {
-  background-color: #df6666;
-  color: #fff;
+.alert-msg--single {
+  font-size: 18pt;
+  font-family: 'Roboto';
```

Comment:
Sounds like more of a Chris question, does the use case/context justify the diversion from standard titles... it might well.

You can use the variables even if they're custom sizes 
https://github.com/nib-styles/v2-typography/blob/master/index.scss#L3-L7